src/components/Select/Select.tsx

```diff
@@ -1,15 +1,22 @@
 import * as React from 'react';
 import classnames from 'classnames';
-
 import { px, useNormalizedInputProps } from '../../utils';
-
 import { InputProps } from '../Input/Input';
+import { SelectVariants } from './types';
 
 export interface SelectProps extends InputProps {
   /**
    * Option elements that are selectable
    */
   children: React.ReactNode;
+  /**
+   * Determines if you want to show the icon
+   */
+  showCustomIcon?: boolean;
```

Comment:
@constantinehuzenko  I still think this this component should be opinionated and force the custom icon to be used (since otherwise the UX is wrong)

Reply:
I don't think there's a usecase where a consumer would want to use the default `<select` appearance for the caret